src/components/AddButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+function AddButton() {
+  let history = useHistory();
+
+  const handleClick = () => {
+    history.push('/add');
+  };
+
+  return (
+    <div className="empty-list">
+      <p>Your shopping list is currently empty.</p>
+      <button className="add-item-btn" onClick={handleClick}>
+        Add Item
+      </button>
```

Comment:
So this a very minor thing but I'm just curious 😅, why did you prefer a button instead of a Link here? I mean, was it just to use the same style of the button in the AddItem component or maybe something else?

Reply:
Maybe you can add the anchor tag inside of the button 🤔 but it won't make any difference though

Reply:
This is an interesting discussion. I just did a quick research and thought this [article](https://formidable.com/blog/2014/05/08/anchors-buttons-and-accessibility/) had great points in terms of using `<button>` and `<a>`.

Reply:
I also just used Accessibility Insights for Web extension and our button can be clicked using the `space` and `enter` keyboard. 👍

Reply:
That's a very good article @Lukabaramishvili Thanks for sharing